project/SotaBuild.scala

```diff
@@ -89,6 +91,18 @@ object SotaBuild extends Build {
       testOptions in RandomTests += Tests.Argument(TestFrameworks.ScalaTest, "-n", "RandomTest"),
       parallelExecution in Test := true,
       dockerExposedPorts := Seq(8081),
+      dockerCommands := dockerCommands.value.filterNot {
```

Comment:
We don't--we remove the default ENTRYPOINT and CMD commands because we're replacing them, and we remove the `USER daemon` command because we need to run the layer that grabs the wait_for_it script before we switch user. (It gets added back in after the wait_for_it script, it's just that there's no 'insert line in the middle' functionality).